services/submission/src/main/java/app/coronawarn/server/services/submission/checkins/EventCheckinDataFilter.java

```diff
@@ -66,7 +66,9 @@ boolean filterOutFutureCheckins(CheckIn checkin) {
   }
 
   boolean filterByValidSignature(CheckIn checkin) {
-    return traceLocationSignatureVerifier.verify(checkin.getSignedLocation());
+    // TODO - @EugenM-SAP do we need still to verify the checkin location?
+    return true;
+    // return traceLocationSignatureVerifier.verify(checkin.getLocationId());
```

Comment:
@EugenM-SAP / @FelixRottler - can you please double check if we still need this or how we need to modify, since there is no 'event-reg' anymore

Reply:
We don't need that part so i will adapt the code.